concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -71,7 +71,9 @@ EOF
 generate_manifest_file() {
   # This exists because concourse does not support boolean value interpolation by design
   enable_auto_deploy=$([ "${ENABLE_AUTO_DEPLOY:-}" ] && echo "true" || echo "false")
+  [ "${DISABLE_CONTINUOUS_SMOKE_TESTS}" = "true" ] && continuous_smoke_tests_trigger="false" || continuous_smoke_tests_trigger="true"
```

Comment:
We chose that form because it only allows value "true" to make it true. Any other value makes it false.
The previous form makes it true whatever the value, as long as the variable is not empty.